src/components/_global/BalChip/composables/useChipStyles.ts

```diff
@@ -18,6 +18,7 @@ export function useChipClasses(props) {
     if (props.color === 'white') return 'bg-white dark:bg-gray-900';
     if (props.color === 'red') return 'text-white bg-red-500';
     if (props.color === 'orange') return 'text-black bg-orange-300';
+    if (props.color === 'orange-500') return 'text-black bg-orange-500';
```

Comment:
```suggestion
    if (props.color === 'orange') return 'text-black bg-orange-500';
```

Reply:
this will affect `new` pools tag as well. is it ok? @pkattera

Reply:
Sorry didn't realise there was already an orange, lets change the names as suggested above.